script.js

```diff
@@ -9,3 +9,169 @@ document.querySelectorAll('.hide-menu').forEach((element) => {
     mobileMenu.style.display = 'none';
   };
 });
+
+const portfolioObjects = {
```

Comment:
This could better be represented as an array and it is also a requirement for the project but I know you have already learned this and I reviewed this late so this is okay 👍

Reply:
Ok, cool i changed it, its fairly simple to do